client/pages/editProject.jsx

```diff
@@ -0,0 +1,378 @@
+import {
+  Button,
+  Card,
+  CardGroup,
+  CardImg,
+  Col,
+  Container,
+  Input,
+  Row,
+  Dropdown,
+  DropdownItem,
+  DropdownToggle,
+  DropdownMenu,
+  Jumbotron,
+  Alert,
+  Carousel,
+  CarouselIndicators,
+  CarouselControl,
+  CarouselCaption,
+  CarouselItem,
+  FormGroup,
+  Label
+} from "reactstrap";
+import {
+  deleteForm,
+  editModel,
+  addModel,
+  getModelsByID,
+  getModels
+} from "../utils/apiWrapper";
+import "../public/styles/home.scss";
+import "../public/styles/style.scss";
+import { Head, Stage } from "../components";
+import { useState } from "react";
+export default function() {
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+  const [title, setTitle] = useState("Loading");
+  const [description, setDescription] = useState("Loading");
+  //Stages
+  let ideationStages = [
+    ["Stage 1", "Description 1"],
+    ["Stage 2", "Description 2"],
+    ["Stage 3", "Description 3"]
+  ];
+  console.log(ideationStages);
+  let addStage = () => {
+    ideationStages.push(["Added Stage Title", "Added Stage Description"]);
+    console.log(ideationStages);
+  };
+  //Dropdown
+  const toggle = () => setDropdownOpen(prevState => !prevState);
+  //Set Title
+  const getTitle = async () => {
+    const project = await getModelsByID("5e5f20e2f182430b94c8c66d");
+    return project.data[0].name;
+  };
+  getTitle().then(name => {
+    name = setTitle(name);
+  });
+  //Set Description
+  const getDescription = async () => {
+    const project = await getModelsByID("5e5f20e2f182430b94c8c66d");
+    return project.data[0].description;
+  };
+  getDescription().then(description => {
+    description = setDescription(description);
```

Comment:
Do you need to assign to `description` here or can you just call `setDescription(description)`?